projects/plugins/jetpack/sal/class.json-api-site-jetpack.php

```diff
@@ -631,4 +631,12 @@ public function get_anchor_podcast() {
 		return $this->get_atomic_cloud_site_option( 'anchor_podcast' );
 	}
 
+	/**
+	 * Get user interactions with a site. Not used in Jetpack.
+	 *
+	 * @return null
+	 */
```

Comment:
```suggestion
	 * Get user interactions with a site. Not used in Jetpack.
	 *
	 * @see /wpcom/public.api/rest/sal/trait.json-api-site-wpcom.php.
	 *
	 * @return null
	 */
```

Minor suggestion, just to keep this comment in line with the others in terms of being able to trace the WPcom file in question better.

Reply:
Thanks, good point, I've added the comment.